### DISCUSSION TITLE
Progress bar chart

### DISCUSSION THREAD

**Discussion by teja-goud-kandula** [apache/superset#22725] [Ideas]
There is guage chart available in super set, 
but it would be sometimes helpful if we can incorporate progress bar chart into the dashboard. 

When targets are a priority of a dashboard, then progress bar can fill the top of the dashboard from left to right. Where as guage chart will fit into a corner, because it takes square shape instead of being rectangular. 

So I request to incorporate progress bar into the charts list. 

![image](https://user-images.githubusercontent.com/16101926/212378309-63925029-cc05-45ae-8169-6786213eae2e.png)

**Comment by loktar00** [comment]
This is a great idea! Something I need to do right now actually... going to have to figure out a creative way to do it otherwise.

**Reply by josephputhumana** [reply]
Hey did you figure out a creative way to do it ?

**Reply by loktar00** [reply]
Unfortunately no :/ I just went with the gauge for my needs
